### PR TITLE
update the hazelcast example to API version v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
update the hazelcast example to API version v1beta3

### CODE REVIEW

**Comment by caesarxuchao** [comment]
@nikhiljindal

File: examples/hazelcast/hazelcast-controller.yaml [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+      containers: 
+        - resources:
+            limits:
+              cpu: 1
+          image: pires/hazelcast-k8s
+          name: hazelcast
+          ports: 
+            - containerPort: 5701
+              name: hazelcast
+
```
Comment (nikhiljindal) [inline examples/hazelcast/hazelcast-controller.yaml]: nit: extra blank line not required

**Comment by caesarxuchao** [comment]
The example is not working properly as stated in #7731, no matter whether I use v1beta1 yaml file or v1beta3 yaml file. It seems to be a problem with the java code of hazelcast rather than Kubernetes.

**Comment by nikhiljindal** [comment]
Thanks @caesarxuchao LGTM.

**Comment by nikhiljindal** [comment]
Ref https://github.com/GoogleCloudPlatform/kubernetes/issues/6584

**Pull request merged**